### ISSUE TITLE
Cannot login with user and password 

### ISSUE THREAD

**Issue by theRAGEhero** [YunoHost-Apps/n8n_ynh#13]
Hello, I've just installed n8n on YunoHost 4.3.5 (stable) but I cannot login.
I specified which username has to be the admin and the admin password but it doesn't allow me to login. 

![image](https://user-images.githubusercontent.com/39453005/150405129-7edf3218-bef0-4ff2-8168-6b64ce396db3.png)

Can you please help me? 
Thank you

**Comment by fflorent** [comment]
Hello there!

Try to authenticate using the user and the password you selected while installing the app.

If you don't remember what you selected, you can check out in the `/var/www/n8n/.env` file the values of `N8N_BASIC_AUTH_USER` and `N8N_BASIC_AUTH_PASSWORD`.

**Comment by theRAGEhero** [comment]
Thank you for your reply.

I fixed it using a subdomain instead of domain.com/n8n.

**Comment by fflorent** [comment]
May I close the issue then?

**Comment by tituspijean** [comment]
There may be an issue with subpath installations then. May be with investigating? 😇

**Comment by nathanael-h** [comment]
Hello, I installed in / and I can not login. Even with the user et pwd from .env file. I tried with both public and private app. I always end to this 401 json error page.

**Comment by nathanael-h** [comment]
I installed from command line. Normally it should'nt be different than with GUI. But did you noticed differences?

**Comment by Krusty27** [comment]
Hi, this is still happening, both with subdomain and subpath (I installed from GUI). What to do?

**Comment by nathanael-h** [comment]
Hello I am facing this issue with an installation from testing, on a dedicated subdomain. Do you have any idea ? The app was installed with public access.

**Comment by guizmoau** [comment]
Installed on a Raspberry Pi4B 8G
Yunohost version 11.0.9.7 (stable)

I had the same issue at first with password for first login

After reading .env file it was the character "&" in my password that was not good..

With a password without "&" it works !

**Comment by zunami** [comment]
I have the same problem

Yunohost 11.0.9.11
n8n 0.191.0~ynh1

it opens username and password window.

it does not work the selected user with the "admin" status.
DOES NOT WORK username and yunohost password
DOES NOT WORK username and n8n admin password
DOES NOT WORK admin and yunohost password
DOES NOT WORK admin and n8n admin password

**Comment by nathanael-h** [comment]
Hello, I manage to connect to n8n and to create the first user, the owner. To do this I disabled the basic auth and made the n8n app public. 
`sudo nano /var/www/n8n/.env` 
`N8N_BASIC_AUTH_ACTIVE=false` 
Then restart: `systemctl restart n8n` 

As a user management exists in the app, and conflicts with the basic auth (cf. last § in https://docs.n8n.io/hosting/user-management/#best-practices ) I suggest we disable the basic auth. What do you think @fflorent and @ericgaspar ?

**Comment by jose1711** [comment]
Disabling basic auth did it for me too. To do it programatically from `root` console session:
```
sed -i '/N8N_BASIC_AUTH_ACTIVE/s/true/false/' /var/www/n8n/.env
systemctl restart n8n
```

**Comment by artybdrlt** [comment]
Same issue here!
Impossible for me to login using the credentials provided during the yunohost cli installation; Jose1711 solution was good. Would it be possible to update the package with a fix or at least put a disclaimer so we are not surprised by this issue? Thanks!

**Comment by theRAGEhero** [comment]
Yeah, it should be the same but it is actually the only thing that I changed and from the GUI it works, I don't know why.

Il 5 aprile 2022 09:53:55 CEST, nathanael-h ***@***.***> ha scritto:
>I installed from command line. Normally it should'nt be different than with GUI. But did you noticed differences?
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/YunoHost-Apps/n8n_ynh/issues/13#issuecomment-1088379931
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>

-- 
Inviato dal mio dispositivo Android con K-9 Mail. Perdonate la brevità.

**Comment by rubensflinco** [comment]
> Olá, consigo me conectar ao n8n e criar o primeiro usuário, o dono. Para fazer isso, desativei a autenticação básica e tornei o aplicativo n8n público. `sudo nano /var/www/n8n/.env` `N8N_BASIC_AUTH_ACTIVE=false` Em seguida, reinicie:`systemctl restart n8n`
> 
> Como existe um gerenciamento de usuários no aplicativo e está em conflito com a autenticação básica (cf. último § em https://docs.n8n.io/hosting/user-management/#best-practices ), sugiro que desativemos a autenticação básica. O que você acha@fflorente@ericgaspar?

Thank you very much for your reply

I opened a pull request asking to disable this by default on all new installs to prevent other newbies like me from having to look around reporting it as a bug.

https://github.com/YunoHost-Apps/n8n_ynh/pull/37

**Comment by wayneoutthere** [comment]
I can confirm this is still the same problem as listed above. I also followed the same advice above as follows and it worked fine:

`sudo nano /var/www/n8n/.env `
`N8N_BASIC_AUTH_ACTIVE=false`
`systemctl restart n8n`

Definitely it would be appreciated if we could get this fix worked into the default deply.  Thanks so much for your efforts here! This is a wonderful app!
